TestFoundation/TestNSArray.swift

```diff
@@ -46,7 +46,10 @@ class TestNSArray : XCTestCase {
             ("test_mutableCopying", test_mutableCopying),
             ("test_writeToFile", test_writeToFile),
             ("test_initWithContentsOfFile", test_initWithContentsOfFile),
-            ("test_readWriteURL", test_readWriteURL)
+            ("test_readWriteURL", test_readWriteURL),
+            ("test_insertObjectAtIndex", test_insertObjectAtIndex),
+            ("test_insertObjectsAtIndexes", test_insertObjectsAtIndexes),
+            ("test_replaceObjectsAtIndexesWithObjects", test_replaceObjectsAtIndexesWithObjects)
```

Comment:
If you add a comma at the end of the list, you avoid the need to do changes like in line 49.

Reply:
@alblue should I update this?

Reply:
You didn't necessarily have to, and I would have squashed the commit with this one rather than appending - but it is something to know for the future, if nothing else :)

Reply:
I avoid sqashing / ammending or changing history in any other way on anything that's open. It's harder to keep track of subsequent changes and comments loose track of the code.

Reply:
Can you squash these three commits together? The first caused a test failure and the comma one standalone doesn't really benefit being on its own.

Reply:
Yup, makes sense to me to do so after review.
Done